openedx/core/djangoapps/discussions/serializers.py

```diff
@@ -406,7 +409,10 @@ def to_representation(self, instance: CourseDiscussionSettings) -> dict:
             'divided_course_wide_discussions': divided_course_wide_discussions,
             'always_divide_inline_discussions': instance.always_divide_inline_discussions,
             'division_scheme': instance.division_scheme,
-            'available_division_schemes': available_division_schemes(course_key)
+            'available_division_schemes': available_division_schemes(course_key),
+            'reported_content_email_notifications': instance.reported_content_email_notifications,
+            'reported_content_email_notifications_flag':
+                REPORTED_CONTENT_EMAIL_NOTIFICATIONS_FLAG.is_enabled(course_key),
```

Comment:
Serializer doesn't look like a good place to add flag it can be moved in view other then that looks good

Reply:
Added the flag here within the `to_representation` method to avoid unnecessary complexity in code.